### ISSUE TITLE
UI RepositoryTestCase on RHEL6 should omit 'ostree'

### ISSUE THREAD

**Issue by lpramuk** [SatelliteQE/robottelo#4154]
Connected to #4040

Test to fix:
- [ ] [UI] RepositoryTestCase.test_negative_download_policy_displayed_for_non_yum_repo

Automation 6.2 results:
```

        if not matched:
>           raise NoSuchElementException("Could not locate element with visible text: %s" % text)
E           NoSuchElementException: Message: Could not locate element with visible text: ostree
```

**Comment by lpramuk** [comment]
```
        non_yum_repo_types = [
            repo_type for repo_type in REPO_TYPE.values()
            if repo_type != 'yum'
        ]
```
```non_yum_repo_types``` or after all yum_repo_types shouldn't contain 'ostree' for RHEL6 runs.

**Comment by lpramuk** [comment]
related to https://github.com/SatelliteQE/robottelo/issues/4040

**Comment by sghai** [comment]
handled by #4183 #4182

**Comment by rochacbruno** [comment]
Fixed in #4183 and #4182
